pkg/kubernetes/upgrade.go

```diff
@@ -169,7 +169,12 @@ func upgradeChartValues(ca, issuerCert, issuerKey string, haMode, mtls bool, arg
 
 func isDowngrade(targetVersion, existingVersion string) bool {
 	target, _ := version.NewVersion(targetVersion)
-	existing, _ := version.NewVersion(existingVersion)
-
+	existing, err := version.NewVersion(existingVersion)
+	if err != nil {
```

Comment:
check if existing is `edge` version ... or `nightly` version ... Then fail ...

Reply:
@mukundansundar  What if the existing installation is locally built say with version `dev-linux-amd64`(as mentioned in the issue) or any other version . Here also the upgrade will fail because the `version.NewVersion()` won't be able to parse.

Reply:
@pravinpushkar Then the output log to the console should be different ....

Reply:
@mukundansundar  Can we go with any one of these - 
1. `fmt.Sprintf("Upgrade failed, %s. The current installed version does not have sematic versioning", err.Error()))`
2. `fmt.Sprintf("Upgrade failed, %s", err.Error()))`